lib/errors.js

```diff
@@ -185,4 +185,29 @@ errors.GatewayTimeoutError = function(message) {
 };
 util.inherits(errors.GatewayTimeoutError, errors.SmartcarError);
 
+/**
+ * Enhanced errors from API v2.0
+ *
+ * @param {Object|String} error - response body from a v2.0 request
+ */
+class SmartcarErrorV2 extends errors.SmartcarError {
```

Comment:
```suggestion
errors.SmartcarErrorV2 = class extends errors.SmartcarError {
```
This should hopefully fix the rendering of the JSDoc (it's currently stating that `SmartcarErrorV2 is an inner prop)